tests/func/test_remote.py

```diff
@@ -145,6 +145,12 @@ def test(self):
         self.assertEqual(default, None)
 
 
+def test_show_default(caplog):
```

Comment:
Don't remove
```suggestion
def test_show_default(dvc_repo, caplog):
```
as it makes the test run in the repo itself instead of in the test env. I think you've removed it because DS was complaining, but it was wrong there 🙂

Reply:
That's right. 😄 Some tool said something about prepending the argument with _unused_ or something like that, but I can't find the message anymore. So should I just keep the argument and let the build fail?

Reply:
@kaiogu Yes, DS will probably fail, but the tests themselves (run on travis) won't, so we are fine 🙂

Reply:
Ill accept after this `dvc_repo` is reintroduced. :)

Reply:
@pared Done :)